# Cancer_Drug_Prediction
 A machine learning and deep learning project for RNA seq data anaysis and drug response prediction
 Data was refered to datasets accompanies the ENLIGHT article listed below[1], RNA-seq data from different files are merged and name of geens have been fetched to the  NCBI Gene database, patients responses to drug are matched. 

[![License](https://img.shields.io/badge/license-MIT-blue.svg)](LICENSE)

---

## Table of Contents

1. [About the Project](#about-the-project)
2. [Getting Started](#getting-started)
3. [Usage](#usage)
4. [Features](#features)
5. [Contributing](#contributing)
6. [License](#license)
7. [Acknowledgments](#acknowledgments)

---

## About the Project

Briefly explain:
- PCA reducing dimensions 
- Visulizing cancer cell amount in each sub cancer type.
- Transcription enrichment in different pathways.
- Gene transcription level in single cells and amount cancer subtypes.

---

## Getting Started

Step-by-step instructions to get a local copy running.

### Prerequisites

List tools and dependencies needed:
- Python 3.10+
- pip install scanpy
- PCA
- umap

### Installation

Provide clear installation steps:

```bash
pip install scanpy

# Navigate to the project directory
cd repo

# Install dependencies

```
### Load the file and Handling Missing values

### Data Visualization

## Contributing

Contributions are welcome! Please follow these steps:

1. Fork the repository.
2. Create your feature branch (`git checkout -b feature/new-feature`).
3. Commit your changes (`git commit -m 'Add new feature'`).
4. Push to the branch (`git push origin feature/new-feature`).
5. Open a pull request.

Refer to [CONTRIBUTING.md](CONTRIBUTING.md) for detailed guidelines.

---

## License

Distributed under the MIT License. See `LICENSE` for more information.

## Citation
[1] Dinstag, G. et al. Clinically oriented prediction of patient response to targeted and immunotherapies from the tumor transcriptome. Med 4, 15â€“30 (2023)

